lib/portage/util/digraph.py

```diff
@@ -1,6 +1,11 @@
 # Copyright 2010-2014 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Type annotation imports
```

Comment:
Please guard this with if type checking

Reply:
Seems tests set `TYPE_CHECKING` to `False`. What to do @thesamesam ?